### PR TITLE
DOC: Add examples for a few Bessel functions

### PR DESCRIPTION
**Pull request by dschmitz89**
#### What does this implement/fix?

Add examples for `iv`, `jv`, `yn` and `yv`.

Examples for the common Bessel functions are done in #16950

[skip azp] [skip actions]

### CODE REVIEW

**Comment by dschmitz89** [comment]
Turns out that I am not that knowledgable about the exponentially scaled Bessel functions such as `yve` (only useful for complex arguments), so restricting the PR to 'iv', 'jv' and ''yv/yn'.

File: scipy/special/_add_newdocs.py [hunk 1]
```diff
@@ -7079,6 +7131,56 @@ def add_newdoc(name, doc):
...
+    >>> jv(0, 1.)
+    0.7651976865579666
+
+    Evaluate the function at one point for different orders.
+
+    >>> jv(0, 1.), jv(1, 1.), jv(1.5, 1.)
+    (0.7651976865579666, 0.44005058574493355, 0.24029783912342725)
+
+    The evaluation for different orders can be carried out in one call by
+    provinding a list or numpy array as argument for the `v` parameter:
```
Comment (tupui) [inline scipy/special/_add_newdocs.py:7149]:
```suggestion
    providing a list or NumPy array as argument for the `v` parameter:
```

File: scipy/special/_add_newdocs.py [hunk 2]
```diff
@@ -11372,11 +11474,61 @@ def add_newdoc(name, doc):
...
+    0.08825696421567697
+
+    Evaluate the function at one point for different orders.
+    >>> yn(0, 1.), yn(1, 1.), yn(2, 1.)
+    (0.08825696421567697, -0.7812128213002888, -1.6506826068162546)
+
+    The evaluation for different orders can be carried out in one call by
+    provinding a list or numpy array as argument for the `v` parameter:
+    >>> yn([0, 1, 2], 1.)
+    array([ 0.08825696, -0.78121282, -1.65068261])
```
Comment (tupui) [inline scipy/special/_add_newdocs.py:11500]:
```suggestion
    The evaluation for different orders can be carried out in one call by
    providing a list or NumPy array as argument for the `v` parameter:

    >>> yn([0, 1, 2], 1.)
    array([ 0.08825696, -0.78121282, -1.65068261])
```

File: scipy/special/_add_newdocs.py [hunk 3]
```diff
@@ -11372,11 +11474,61 @@ def add_newdoc(name, doc):
...
+    Examples
+    --------
+    Evaluate the function of order 0 at one point.
+
+    >>> from scipy.special import yn
+    >>> yn(0, 1.)
+    0.08825696421567697
+
+    Evaluate the function at one point for different orders.
+    >>> yn(0, 1.), yn(1, 1.), yn(2, 1.)
```
Comment (tupui) [inline scipy/special/_add_newdocs.py:11494]:
```suggestion
    Evaluate the function at one point for different orders.

    >>> yn(0, 1.), yn(1, 1.), yn(2, 1.)
```

File: scipy/special/_add_newdocs.py [hunk 4]
```diff
@@ -11419,13 +11571,63 @@ def add_newdoc(name, doc):
...
+    Examples
+    --------
+    Evaluate the function of order 0 at one point.
+
+    >>> from scipy.special import yv
+    >>> yv(0, 1.)
+    0.088256964215677
+
+    Evaluate the function at one point for different orders.
+    >>> yv(0, 1.), yv(1, 1.), yv(1.5, 1.)
```
Comment (tupui) [inline scipy/special/_add_newdocs.py:11592]:
```suggestion
    Evaluate the function at one point for different orders.

    >>> yv(0, 1.), yv(1, 1.), yv(1.5, 1.)
```

File: scipy/special/_add_newdocs.py [hunk 5]
```diff
@@ -11419,13 +11571,63 @@ def add_newdoc(name, doc):
...
+    (0.088256964215677, -0.7812128213002889, -1.102495575160179)
+
+    The evaluation for different orders can be carried out in one call by
+    provinding a list or numpy array as argument for the `v` parameter:
+    >>> yv([0, 1, 1.5], 1.)
```
Comment (tupui) [inline scipy/special/_add_newdocs.py:11597]:
```suggestion
    providing a list or NumPy array as argument for the `v` parameter:

    >>> yv([0, 1, 1.5], 1.)
```

**Review by tupui — CHANGES_REQUESTED** [review]
Thanks @dschmitz89, some small things to tweak.

**Comment by dschmitz89** [comment]
@tupui : refguide is passing and all style issues resolved. Anything else missing?

**Review by tupui — APPROVED** [review]
LGTM, thanks again for the PR (and the ping) @dschmitz89

**Pull request merged**